src/js/smtpserver.c

```diff
@@ -1,19 +1,154 @@
 #include "smtpserver.h"
 
+jsval create_response(JSContext *cx, int code, const char* message, int disconnect) { 
+	jsval rcode;
```

Comment:
Suggest using the direct value in `JS_DefineProperty()` below and removing the intermediate variable. Same for `rdisconnect`.